Configure

```diff
@@ -672,8 +665,7 @@ while ((my $first, my $second) = (shift @list, shift @list)) {
 # The default is the Unix like separator, :, but as an exception, we also
 # support the space as separator.
 my $list_separator_re =
-    { VMS           => qr/(?<!\^),/,
-      MSWin32       => qr/(?<!\\);/ } -> {$^O} // qr/(?<!\\)[:\s]/;
+    { MSWin32       => qr/(?<!\\);/ } -> {$^O} // qr/(?<!\\)[:\s]/;
```

Comment:
I would fix the style to put the `= {` but since this goes away for cmake, don't bother :)